pkg/provisioners/provisioner_util.go

```diff
@@ -69,13 +78,23 @@ func newDispatcherService(ccp *eventingv1alpha1.ClusterChannelProvisioner) *core
 			Selector: labels,
 			Ports: []corev1.ServicePort{
 				{
-					Name:       "http",
+					// There is a bug in Istio where named port doesn't work when connecting using an ExternalName service
+					// Refer to https://github.com/istio/istio/issues/13193 for more details.
+					// TODO: Revert this when ISTIO fixes the issue
```

Comment:
Revert what? I would leave the `Name: "http",` line in the code, but comment it out and have this TODO say to add the line back when the bug is fixed.